.gitignore

```diff
@@ -1,21 +1 @@
-*.gem
```

Comment:
Looks good, but were these accidentally removed?

Reply:
No, intentionally. Any reason to keep those?

Reply:
I don't think we need to keep most of these, but I was thinking we still needed `.gem` in here because they are created when making new releases. However, it looks like github actions can now take care of publishing new gems, so yeah I don't see any reasons to keep those. 👍